compatibility/tests/integration-transports/src/test/java/org/mule/compatibility/config/spring/parsers/specific/ComponentWithBindingsDefinitionParserFlowTestCase.java

```diff
@@ -31,13 +31,13 @@
 import org.mule.runtime.core.api.context.MuleContextFactory;
 import org.mule.runtime.core.component.AbstractJavaComponent;
 import org.mule.runtime.core.context.DefaultMuleContextFactory;
-import org.mule.runtime.core.model.resolvers.ArrayEntryPointResolver;
-import org.mule.runtime.core.model.resolvers.CallableEntryPointResolver;
-import org.mule.runtime.core.model.resolvers.DefaultEntryPointResolverSet;
-import org.mule.runtime.core.model.resolvers.ExplicitMethodEntryPointResolver;
-import org.mule.runtime.core.model.resolvers.MethodHeaderPropertyEntryPointResolver;
-import org.mule.runtime.core.model.resolvers.NoArgumentsEntryPointResolver;
-import org.mule.runtime.core.model.resolvers.ReflectionEntryPointResolver;
+import org.mule.runtime.core.api.model.resolvers.ArrayEntryPointResolver;
```

Comment:
Do these need to be in API in order to be used by XML parser? Otherwise why are these API?  Also these may need to go in compatibility module after further cleanup.